dev/how-to/deploy/data-migration-with-binary.md

```diff
@@ -0,0 +1,367 @@
+
+```toml
+# Worker Configuration.
+
+# log configuration
+log-level = "info"
+log-file = "dm-worker.log"
+
+# DM-worker address
+worker-addr = ":8262"
+
+# The server ID of MySQL slave, used when pulling binlog from MySQL
+# In a replication group, each instance (master and slave) must have a unique server ID
+server-id = 101
+
+# Used to mark a replication group or MySQL/MariaDB instance
+source-id = "mysql-replica-01"
+
+# The type of upstream instance
+# Available values: "mysql", "mariadb"
+flavor = "mysql"
+
+# The path in which relay logs are stored
+relay-dir = "./relay_log"
+
+# The path in which the metadata of DM-worker is stored
+meta-dir = "dm_worker_meta"
+
+# Enables or disables gtid in the relay log unit
+enable-gtid = false
+
+# The charset configuration in the DSN for connecting MySQL or Mariadb
+# charset= ""
+
+# MySQL connection address
+[from]
+host = "192.168.0.1"
+user = "root"
+password = "fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg="
+port = 3306
+
+# The rules for deleting relay logs
+# [purge]
+# interval = 3600
+# expires = 24
+# remain-space = 15
+
+# Configurations of task status check
+# [checker]
+# check-enable = true
+# backoff-rollback = 5m
+# backoff-max = 5m
+```
+
+Then, run the following command in the terminal to operate DM-worker:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+bin/dm-worker -config conf/dm-worker1.toml
+```
+
+In DM-worker2, change `source-id` in the configuration file to `mysql-replica-02` and change the `from` configuration part to the address of MySQL2.
+
+### Deploy DM-master
+
+You can configure DM-worker by using command-line parameters or the configuration file.
+
+**Deployment method 1: DM-master command-line parameters**
+
+Below is the description of DM-master command-line parameters:
+
+```bash
+./bin/dm-master --help
+```
+
+```
+Usage of dm-master:
+  -L string
+        Log level. Available values: "debug", "info" (default value), "warn", "error" or "fatal"
+  -V    The output version
+  -config string
+        The path of the configuration file
+  -log-file string
+        The path of log files
+  -master-addr string
+        DM-master address
+  -print-sample-config
+        Prints the example configuration of DM-master
+```
+
+> **Note:**
+>
+> In some situations, you cannot use the above method to configure DM-master because some configurations are not exposed to the command line. Then use the configuration file instead.
+
+**Deployment method 2: configuration file**
+
+Below is the configuration file of DM-master. It is recommended that you use this method and write the following configuration to `conf/dm-master.toml`.
+
+```toml
+# Master Configuration.
+
+# RPC configurations
+rpc-rate-limit = 10.0
+rpc-rate-burst = 40
+
+# Log configurations
+log-level = "info"
+log-file = "dm-master.log"
+
+# The listening address of DM-master
+master-addr = ":8261"
+
+# replication group <-> DM-worker configuration
```

Comment:
```suggestion
# Replication group <-> DM-worker configuration
```
This `<->` symbol means "refers to DM-worker configuration section"?

Reply:
I have confirmed with @WangXiangUSTC  that it is a typo.